stdlib/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 # Create convenience targets for the Swift standard library.
+
+set(CMAKE_CXX_COMPILER "${SWIFT_NATIVE_LLVM_TOOLS_PATH}/clang++")
+set(CMAKE_C_COMPILER "${SWIFT_NATIVE_LLVM_TOOLS_PATH}/clang")
```

Comment:
We can't use the just-built clang when building the official toolchain for Xcode.  We are only allowed to use the Xcode's clang.  If we use an arbitrary clang, bitcode workflows will be broken.